client/tools/mgr-virtualization/scripts/Makefile

```diff
@@ -1,11 +1,9 @@
 PKGDIR0 = /etc/rc.d/init.d
-PKGDIR1 = /etc/cron.d
 SYSCONF_DIR = /etc/sysconfig/rhn
 CONF_DIR = /etc/sysconfig/rhn/virt
 CONF_DIR_AUTO = /etc/sysconfig/rhn/virt/auto
 
 SCRIPTS = rhn-virtualization-host
-OTHER = rhn-virtualization.cron
```

Comment:
If we remove the cron stuff from here, I think the package will not be able to build anymore for other than python3 distributions, am I right?

Reply:
The plan was to have it still for both. We still need it for SLE12.

Reply:
This gets installed with the spec now, right?